lighthouse-core/gather/gather-runner.js

```diff
@@ -110,14 +110,14 @@ class GatherRunner {
   }
 
   /**
-   * Catches any `recoverable` errors from the supplied promise, rejecting on
-   * the rest.
+   * Test any error output from the promise, absorbing non-fatal errors and
+   * throwing on fatal ones so that run is stopped.
    * @param {!Promise<*>} promise
    * @return {!Promise<*>}
    */
   static recoverOrThrow(promise) {
     return promise.catch(err => {
-      if (!err.recoverable) {
+      if (err.fatal) {
```

Comment:
Any of the same machinery for audits? Or are we still treating all errors there as fatal?